src/utils/types.ts

```diff
@@ -1,3 +1,14 @@
 import type { Ref } from "vue";
 
 export type MaybeRef<T> = T | Ref<T>;
+
+export interface JwtToken {
```

Comment:
I first went for GoogleUser but then went back to this because I thought the developer would anyway reuse the content on his own to fill some kind of structure on his side.
Technically, it is a decoded jwt token but I thought it was too long.

To be honest, it's your call: I'm don't care much how its named as long as its content is clear to me ;)
Let me know what you prefer.

Reply:
`DecodedGoogleUser` seems also nice for me, since this is very domain specific...most users will ask whats this JWT thing again. Lets go with 

`decodeCredential` and `DecodedGoogleUser` 
I prefer it more than JWT